src/html_parser/newsplease.py

```diff
@@ -140,8 +140,6 @@ def _newsplease_article_to_parsed_html(
             document_content_type=input.document_content_type,
             document_slug=input.document_slug,
             html_data=HTMLData(
-                detected_title=newsplease_article.title,
```

Comment:
Just confirmed by searching the cpr codebase in github and I don't think these are used anywhere.

Reply:
yeah, they aren't! this was me being ambitious when originally writing this and wondering whether we could automatically set document titles using detected titles from HTMLs